doc/nrf/external_comp/dult.rst

```diff
@@ -244,6 +249,41 @@ Asynchronous support is also necessary to report sound state changes that are tr
 
 .. rst-class:: numbered-step
 
+.. _ug_dult_motion_detector:
+
+Interacting with the motion detector
+************************************
+
+DULT motion detector is an optional feature of the DULT subsystem.
+For more details, see the `DULT Motion detector`_ section of the DULT documentation.
+You can enable the :kconfig:option:`CONFIG_DULT_MOTION_DETECTOR` Kconfig option to support the DULT motion detector feature in your project.
+
+To integrate the motion detector feature, set the motion detector unwanted tracking accessory capability bit (:c:enum:`DULT_ACCESSORY_CAPABILITY_MOTION_DETECTOR_UT_BIT_POS`) in the accessory capabilities bitmask.
+You must do this when registering the DULT user to indicate support for this feature.
+
+To register the motion detector callbacks, use the :c:func:`dult_motion_detector_cb_register` function.
```

Comment:
This part down to the line 283 seems to be the same as in the Fast Pair doc, so you could do some reuse. Check https://github.com/nrfconnect/sdk-nrf/blob/main/doc/nrf/templates/cheat_sheet.rst (Reuse chapter). So in this case, add link tags just before the text to be copied and right after it something like this:

.. _motion_detector_reqs_start

To register...

.. _motion_detector_reqs_end

And then use them like this:

.. include:: ../bt_fast_pair.rst
     :start-after: motion_detector_reqs_start
     :end-before: motion_detector_reqs_end

Reply:
There is a difference in the structure name. One is `dult_motion_detector_cb` and the other one is `bt_fast_pair_fmdn_motion_detector_cb`.

Reply:
Is there a way to substitute it in all places without copying the whole part.

Reply:
You can use the same mechanism up until the part where the difference is found. And right after that, have another pair of link tags, so have two different includes.

Reply:
The difference is present 6 times across 19 lines so I think it would be a mess to do it that way.

Reply:
I agree. Sorry for the hassle, I did not read thoroughly enough. But at least, you now know how to reuse when possible.

Reply:
No problem, It's a shame that you can't use it as a "function" and include it with the parameter that could be changed in each include.